node-repository/src/main/java/com/yahoo/vespa/hosted/provision/restapi/v2/filter/AuthorizationFilter.java

```diff
@@ -31,8 +32,12 @@
     private final BiConsumer<ErrorResponse, ResponseHandler> rejectAction;
 
     @Inject
-    public AuthorizationFilter(Zone zone, NodeRepository nodeRepository) {
-        this.authorizer = new Authorizer(zone.system(), nodeRepository);
+    public AuthorizationFilter(NodeRepository nodeRepository, FilterConfig filterConfig) {
```

Comment:
Don't user FilterConfig - this is a deprecated feature that we want to remove soon. Introduce a new config definition instead.